examples/build.gradle

```diff
@@ -1,17 +1,15 @@
 dependencies {
-    // implementation "com.hedera.hashgraph:sdk:2.0.3"
-    implementation project(":sdk")
-
     implementation "com.google.code.gson:gson:2.10.1"
     implementation "org.slf4j:slf4j-simple:2.0.7"
     implementation "io.grpc:grpc-netty-shaded:1.55.1"
     implementation "io.github.cdimascio:java-dotenv:5.3.1"
     implementation "com.google.errorprone:error_prone_core:2.19.1"
+    implementation project(path: ':sdk')
 }
 
 java {
-    sourceCompatibility = org.gradle.api.JavaVersion.VERSION_1_8
-    targetCompatibility = org.gradle.api.JavaVersion.VERSION_1_8
+    sourceCompatibility = 17
+    targetCompatibility = 17
```

Comment:
Should only need to set these in root `build.gradle` and remove from all sub projects.